.circleci/config.yml

```diff
@@ -8,3 +8,18 @@ workflows:
           name: lint
       - ruby-rails/test-gem:
           name: test
+          before-test:
+            - run:
+                name: Install exiftool
+                command: curl -L http://cpanmin.us | perl - --sudo Image::ExifTool
+            - run:
+                name: Install libvips
+                command: sudo apt-get update && sudo apt-get install -y libvips
+            - setup_remote_docker
+            - run:
+                name: Install Kakadu
+                command: |
+                  docker create --name kakadu josejuansanchez/kakadu:1.0
```

Comment:
This is not an official release channel for this software. This seems to package kakadu 7, which is not the latest release. Version 8 was released 2 years ago. Is this the right choice?

Reply:
I scanned this image and found 122 security vulnerabilities (3 are high severity):
```
 docker scan josejuansanchez/kakadu:1.0
 ```
 
 This is probably not a huge deal in how we plan on using this, but it might not be the best choice.

Reply:
I'm matching https://github.com/sul-dlss/assembly-image/blob/main/.circleci/config.yml.   I see in this PR it was chosen because samvera uses it.  https://github.com/sul-dlss/assembly-image/pull/40

Do you know of a better docker image we could use?  If so, please make a ticket for using a different kakadu in circleci, and I'll get this and assembly-image switched over.

Reply:
Kakadu is very restrictive about distribution of there software.  I think they email you a CDROM or something. I think this is a bootleg.

Reply:
I believe the demo is only good for one year, then you have to pay.

Reply:
This is the same zip (abet an older out of date one) is in the dockerfile that you currently have: https://github.com/josejuansanchez/docker-kakadu/blob/master/Dockerfile#L15

Reply:
switched it.

Reply:
switched it.